docker-envs/base-ror-docker/es/install-ror-es-using-file.sh

```diff
@@ -13,6 +13,6 @@ echo "Installing ES ROR from file..."
 /usr/share/elasticsearch/bin/elasticsearch-plugin install --batch file:///tmp/ror.zip
 if verlte "6.5.0" "$ES_VERSION"; then
   echo "Patching ES ROR..."
-  /usr/share/elasticsearch/jdk/bin/java -jar /usr/share/elasticsearch/plugins/readonlyrest/ror-tools.jar patch
+  /usr/share/elasticsearch/jdk/bin/java -jar /usr/share/elasticsearch/plugins/readonlyrest/ror-tools.jar patch --I-understand-implications-of-ES-patching yes
```

Comment:
In KBN ROR plugin it's [I_UNDERSTAND_AND_ACCEPT_KBN_PATCHING ](https://docs.readonlyrest.com/kibana#patching-kibana-acknowledgment-in-a-silent-mode). Let's make it on the ROR ES side symmetric (and case insensitive)

Reply:
yeah, I agree it looks better but IMO our plugins management should be similar. It would be little surprising for a user.  I'm OK with the approach you proposed as long as you add this in the ROR KBN plugin too (preferably in the backward compatible way ;))